sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/ResourceLabels.java

```diff
@@ -1,7 +1,10 @@
 package com.mesosphere.sdk.offer.evaluate;
 
 import com.mesosphere.sdk.specification.ResourceSpec;
+
 import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.mesos.Protos.ResourceProviderID;
```

Comment:
Should just import `Protos` here and then refer to the types as `Protos.X` below